### PR TITLE
Language changes - README.md, addon.xml, Swedish and Norwegian

### PR DESCRIPTION
**Pull request by Snaah**
**Changes**
- addon.xml - summary, description and disclaimer for English rewritten + Swedish added
- README.md - a lot of changes/corrections
- strings.po (Swedish) - changed "Strömbithastighet" to the correct "Bithastighet" + changes to make context menu strings for Channel Settings show
- chn_sbs.json, chn_tv4se.json, chn_urplay.json and chn_viafree.json - changed to a consistent standard for the channel descriptions  for Norwegian and Swedish + fixed some minor language issue

### CODE REVIEW

File: plugin.video.retrospect/addon.xml [hunk 1]
```diff
@@ -117,11 +117,18 @@
     <extension point="xbmc.addon.metadata">
         <reuselanguageinvoker>true</reuselanguageinvoker>
         <platform>all</platform>
-        <summary lang="en_GB">Retrospect (previously XBMC Online TV) is a plugin that allows the playback of a lot of TV shows that are available (legally) via the broadcasters websites.</summary>
-        <description lang="en_GB">Retrospect uses the official websites of a lot of different broadcasting companies (mainly Dutch, Belgian, British and Swedish) to make their re-run episodes available on the Kodi platform. More information can be found at www.rieter.net.[CR][CR]Retrospect reports back usage statistics to our servers. This option can be disabled from the add-on settings.</description>
-        <disclaimer lang="en_GB">Retrospect is provided as is. We do not take any responsibility for possible (but highly unlikely) damage caused to your system. Nor do we take responsibility for the content that can be watched via this add-on or any consequences of using this add-on to watch the content using the content provider's platforms.</disclaimer>
+
```
[the remark this answers is not in the document]
Comment (Snaah) [reply plugin.video.retrospect/addon.xml]: Found no spaces or tabs on line 120, but removed those on 116. Ok now?

File: plugin.video.retrospect/addon.xml [hunk 2]
```diff
@@ -117,11 +117,18 @@
...
-        <disclaimer lang="en_GB">Retrospect is provided as is. We do not take any responsibility for possible (but highly unlikely) damage caused to your system. Nor do we take responsibility for the content that can be watched via this add-on or any consequences of using this add-on to watch the content using the content provider's platforms.</disclaimer>
+	    
+        <summary lang="en_GB">Retrospect enables playback of TV shows legally available online.</summary>
+        <description lang="en_GB">Retrospect uses the official websites of different broadcasting companies (mainly Dutch, Belgian, British, Norwegian and Swedish) to make their re-run/catchup episodes available on the Kodi platform.[CR][CR]More information can be found at https://rieter.net or the Retrospect wiki at https://github.com/retrospect-addon/plugin.video.retrospect/wiki/.</description>
```
[the remark this answers is not in the document]
Comment (Snaah) [reply plugin.video.retrospect/addon.xml]: Fixed.

File: plugin.video.retrospect/addon.xml [hunk 3]
```diff
@@ -117,11 +117,18 @@
...
         <reuselanguageinvoker>true</reuselanguageinvoker>
         <platform>all</platform>
-        <summary lang="en_GB">Retrospect (previously XBMC Online TV) is a plugin that allows the playback of a lot of TV shows that are available (legally) via the broadcasters websites.</summary>
-        <description lang="en_GB">Retrospect uses the official websites of a lot of different broadcasting companies (mainly Dutch, Belgian, British and Swedish) to make their re-run episodes available on the Kodi platform. More information can be found at www.rieter.net.[CR][CR]Retrospect reports back usage statistics to our servers. This option can be disabled from the add-on settings.</description>
-        <disclaimer lang="en_GB">Retrospect is provided as is. We do not take any responsibility for possible (but highly unlikely) damage caused to your system. Nor do we take responsibility for the content that can be watched via this add-on or any consequences of using this add-on to watch the content using the content provider's platforms.</disclaimer>
+	    
+        <summary lang="en_GB">Retrospect enables playback of TV shows legally available online.</summary>
```
[the remark this answers is not in the document]
Comment (Snaah) [reply plugin.video.retrospect/addon.xml]: Fixed. Makes the Swedish string much shorter, as "playkanaler" itself means it's TV shows online.

File: plugin.video.retrospect/addon.xml [hunk 4]
```diff
@@ -117,11 +117,18 @@
...
+
+        <summary lang="sv_SE">Retrospect möjliggör att visa TV-program som finns lagligt tillgängliga online.</summary>
+        <description lang="sv_SE">Retrospect använder olika TV-kanalers (framförallt nederländska, belgiska, brittiska, norska och svenska) officiella webbsidor för att göra deras playkanaler tillgängliga i Kodi.[CR][CR]Mer information finns på https://rieter.net eller Retrospects wiki på https://github.com/retrospect-addon/plugin.video.retrospect/wiki/.</description>
+        <disclaimer lang="sv_SE">Retrospect tillgängliggörs i befintligt skick. Vi tar inget som helst ansvar för möjliga (men högst osannolika) skador på ditt system. Vi tar inte heller ansvar för innehållet som kan ses med detta tillägg eller konsekvenser av att använda tillägget för att se innehållet via innehållsleverantörernas plattformar.</disclaimer>
+
         <license>CC-BY-NC-SA-4.0</license>
         <language>en nl de se no lt lv fi</language>
+
```
[the remark this answers is not in the document]
Comment (Snaah) [reply plugin.video.retrospect/addon.xml]: Fixed.

File: README.md [hunk 5]
```diff
@@ -5,83 +5,80 @@
...
+Starting from Kodi Leia (v18), you can easily install Retrospect from the official Kodi add-on repository. Simply use the search function in the add-ons section to find `Retrospect` and install it. More detailed information can be found in the [Retrospect Wiki](https://github.com/retrospect-addon/plugin.video.retrospect/wiki/Installation).
 
-**Note:** Make sure that the _Auto-Update_ option for Retrospect is enabled to automagically receive new updates.
+**Note:** Make sure the _Auto-Update_ option for Retrospect is enabled to automagically receive new updates.
 
 ![alt text](./plugin.video.retrospect/resources/media/retrospect01.jpg "The Retrospect information screen")
 
 ### Kodi Krypton    
 If you are running Kodi Krypton (v17) the only way to install Retrospect is to install it from a zip file. Follow these steps (keep in mind they might slightly differ depending on your skin):
-
```
[the remark this answers is not in the document]
Comment (Snaah) [reply README.md]: Just to make sure: There's an empty line before the other lists. Still remove this?

File: README.md [hunk 6]
```diff
@@ -5,83 +5,80 @@
...
 
 The _Easy way_:
 
 1. Open the Retrospect add-on settings.
-1. Enable support for the InputStream Adaptive add-on within Retrospect by enabling `Use Kodi InputStream Adaptive add-on when possible`.
-1. Select **OK** to save the settings.
-1. Open the Retrospect add-on settings again.
-1. Go to `Install Widevine using InputStream Adaptive Helper add-on` en select this option.
-1. Complete the steps that the InputStream Adaptive Helper add-on give you.
+1. On the starting screen, make sure `Use Kodi InputStream Adaptive add-on when possible` is enabled.
```
[the remark this answers is not in the document]
Comment (Snaah) [reply README.md]: Fixed.

File: README.md [hunk 7]
```diff
@@ -5,83 +5,80 @@
...
 1. From that configuration file, find the image for an ARM device that resembles your device. 
-    * Multiple successes have been reported using the the *Acer Chromebook R13* image.
-    * The device configuration section in the config file have an `url` field that contains a link to a recovery image. 
-    * In the case of the *Acer Chromebook R13* you can download the full recovery from this url: [https://dl.google.com/dl/edgedl/chromeos/recovery/chromeos_{version}_elm_recovery_stable-channel_mp.bin.zip](https://dl.google.com/dl/edgedl/chromeos/recovery/chromeos_11021.81.0_elm_recovery_stable-channel_mp.bin.zip).
-1. From that recovery image you will need the Widevine files located in `/opt/google/chrome/libwidevinecdm*.so`.
-1. These files need to be copied to the `<kodi-profile>/cdm` folder.
+    * The *Acer Chromebook R13* image has been reported as working for many devices.
+    * The device configuration section in the config file has a `URL` field that contains a link to a recovery image.
+1.From that recovery image, you will need the Widevine files located in /opt/google/chrome/libwidevinecdm*.so.
```
[the remark this answers is not in the document]
Comment (Snaah) [reply README.md]: Damn, missed those. Fixing.

File: plugin.video.retrospect/addon.xml [hunk 8]
```diff
@@ -145,4 +151,4 @@ Mainly a release to update the Retrospect license from CC BY-NC-ND to CC BY-NC-S
             <screenshot>resources/media/retrospect03.jpg</screenshot>
         </assets>
     </extension>
-</addon>
\ No newline at end of file
+</addon>
```
Comment (Snaah) [inline plugin.video.retrospect/addon.xml]: I've tried numerous times to remove the empty line at the end, both by editing online and locally (before making the PR). Github won't let me!

File: README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (basrieter) [reply README.md]: Don't remove it. It needs to stay.

File: plugin.video.retrospect/addon.xml (hunk 8 above)
Comment (basrieter) [reply plugin.video.retrospect/addon.xml]: @Snaah Don't update your PR via GitHub. Do it locally and then push the changes.

**Comment by basrieter** [comment]
Thanks for updating the PR. Only thing left is that single line-ending thing.

**Pull request merged**